src/service/viewer-impl.js

```diff
@@ -245,14 +246,21 @@ export class Viewer {
         this.prerenderSize_;
     dev().fine(TAG_, '- prerenderSize:', this.prerenderSize_);
 
+    const queryParams = parseQueryString(this.win.location.search);
```

Comment:
We previously had this inlined to avoid unnecessary work when `this.isIframed_ == true`.

Reply:
Updated so that it doesn't do the work when this.isIframed == true